src/components/Technologies/Technologies.test.tsx

```diff
@@ -7,7 +7,7 @@ test('renders Technologies component', () => {
   const technologiesComponent = render(
     <Technologies />
   )
-  const iconsAlt = ['Vite Logo', 'React Logo', 'Typescript Logo', 'Playwright Logo', 'Storybook Logo', 'Hygen Logo', 'Stitches Logo']
+  const iconsAlt = ['Vite Logo', 'React Logo', 'Typescript Logo', 'Playwright Logo', 'Hygen Logo', 'Stitches Logo']
```

Comment:
Should we change the Stitches by the Styled one?

Reply:
Stitches are still present in this PR.